scheduler/kubernetes/upgrade.md

```diff
@@ -8,57 +8,115 @@ sidebar: home_sidebar
-        ```yaml
-        updateStrategy:
-            type: OnDelete
-        ```
-
-### 2. Upgrade the Portworx spec
-
-* Change the image of the Portworx Daemonset
-    * Set the image with command: `$ kubectl set image ds portworx portworx=portworx/px-enterprise:1.2.10 -n kube-system`
-    * Alternately, you can also change the image in the DaemonSet spec file and apply it using `$ kubectl apply -f <px-spec.yaml>`.
-* Update the `ClusterRole` permissions in Portworx spec using below:
-
-    ```
-    cat <<EOF | kubectl apply -f -
-    kind: ClusterRole
-    apiVersion: rbac.authorization.k8s.io/v1alpha1
-    metadata:
-       name: node-get-put-list-role
-    rules:
-    - apiGroups: [""]
-      resources: ["nodes"]
-      verbs: ["get", "update", "list"]
-    - apiGroups: [""]
-      resources: ["pods"]
-      verbs: ["get", "list"]
-    EOF
-    ```
-
-### 3. Upgrade Portworx pods
-
-It is not recommended to delete the Portworx pod while an application is actively issuing I/O. This can induce race conditions in docker causing it to hang. 
-
-The following procedure should be followed:
-1. Cordon the node where you want to upgrade Portworx: `$ kubectl cordon <node-name>`
-2. Delete application pods running on this node that are using Portworx volumes: `$ kubectl delete pod <pod-name>`
-    * Since application pods are expected to be managed by a controller like `Deployement` or `StatefulSet`, Kubernetes will spin up a new replacement pod on another node.
-3. Delete Portworx pod running on this node. This will start a new Portworx pod on this node with the new version you set above. (Note: Cordoning a kubernetes node doesn't affect DaemonSet pods)
-4. A new Portworx pod with the new version will be initiated on this node. This pod will stay in initializing state.
-5. Reboot the node. (This step is needed only if your current Portworx version is 1.2.9 since it requires a reboot of the host to perform upgrade of our kernel driver.)
-6. Uncordon the node once it comes up.
+This guide describes the procedure how to upgrade Portworx in Kubernetes environment as OCI container, which is the default and recommended method of running Portworx in Kubernetes.
+
+We assume you have deployed Portworx as [Kubernetes Daemonset](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) using instructions provided at our [install page](/scheduler/kubernetes/install.html).
+
+
+>**IMPORTANT:**<br/>We do not recommend upgrading Portworx using [Kubernetes instructions](https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/) (e.g. via `kubectl set image ds/portworx portworx=portworx/XXXX:### -n kube-system`).<br/>
+>Instead, whenever possible, we recommend using the [install.portworx.com](http://install.portworx.com) site to generate latest recommended YAML-spec, and re-apply the spec to upgrade Portworx.
```

Comment:
Remove the line "Instead, whenever possible, we recommend using the install.portworx.com site to generate latest recommended YAML-spec, and re-apply the spec to upgrade Portworx." since that is what this page already describes in good detail.